### DISCUSSION TITLE
traces to aff_a

### DISCUSSION THREAD

**Discussion by cuhlig42** [JCluzet/42_EXAM#54] [Q&A]
hey i hope this is the right section for this.
after some try and test i got this in my trace:

a$
a-------
a$
a-------
a$
a-------
$
a-------
a$
a-------
a$
a-------

and i got the message assigment failed 
but im not sure what ive to change now ^^

**Comment by JCluzet** [comment]
Hi @cuhlig42 

Can you send me the exercise you did so I can try to reproduce the same thing on my side? 

The error seems to me to be related to a missing or too much `/n`
